certbot/ocsp.py

```diff
@@ -95,15 +94,20 @@ def determine_ocsp_server(self, cert_path):
 def _translate_ocsp_query(cert_path, ocsp_output, ocsp_errors):
     """Parse openssl's weird output to work out what it means."""
 
-    if not "Response verify OK" in ocsp_errors:
+    pattern = r"{0}: (WARNING.*)?good".format(cert_path)
+    good = re.search(pattern, ocsp_output, flags=re.DOTALL)
+    warning = good.group(1) if good else None
+
+    if (not "Response verify OK" in ocsp_errors) or (good and warning):
         logger.info("Revocation status for %s is unknown", cert_path)
-        logger.debug("Uncertain ouput:\n%s\nstderr:\n%s", ocsp_output, ocsp_errors)
+        logger.debug("Uncertain output:\n%s\nstderr:\n%s", ocsp_output, ocsp_errors)
         return False
-    if cert_path + ": good" in ocsp_output:
+    elif good and not warning:
         return False
     elif cert_path + ": revoked" in ocsp_output:
```

Comment:
we also might get a warning with revoked, see http://osxr.org:8080/openssl/source/apps/ocsp.c#1019